VideoLocker/res/values/dimens.xml

```diff
@@ -24,6 +24,8 @@
     <dimen name="course_detail_margin_right">20dp</dimen>
     <dimen name="course_detail_margin">12dp</dimen>
     <dimen name="default_row_text">12sp</dimen>
+    <dimen name="course_card_height">190dp</dimen>
+    <dimen name="course_detail_card_height">70dp</dimen>
```

Comment:
the mockups pronounce it as `title card`. Plus its a card on a card, so, yeah I think the name is fine